src/constants.js

```diff
@@ -9,6 +9,13 @@ export const POST_MESSAGE = {
   ALLOW_DELEGATE: `${ZOID}_allow_delegate`,
 };
 
+export const RENDER_ERRORS = {
```

Comment:
Nit: all the other constants are singular in this file. It would be nice to follow that same naming convention.
```suggestion
export const RENDER_ERROR = {
```

Reply:
Also, what do you think about the name `COMPONENT_ERROR` instead of `RENDER_ERROR`. I think some of these errors can occur after the render phase is complete.